.github/workflows/mac_python_matrix_test.yml

```diff
@@ -0,0 +1,49 @@
+name: continuous-integration/github/pr/osx
+
+on:
+  pull_request:
```

Comment:
Can you put a `name:` field here to suppress the `(Pull Request)` text in the check name?

Reply:
According to Github Actions documentation, the `on` section doesn't support the `name` field. 

I'm going to open a question with Github to see if there's a way to suppress `(pull_request)` on the Status Checks section. It doesn't show up on the job itself if you open the `Actions` tab.

Reply:
Great.  I am not too worried about it - it was just an effort to make things more concise and uniform (and so that the description would show up when looking at the checks list).

Reply:
As per [the posted question](https://github.community/t5/GitHub-Actions/Suppress-pull-request-from-check-name/m-p/42917/highlight/false#M5173), suppressing is not a capability (pretty much as we expected).